ce/test/core/msbuild-tests.ts

```diff
@@ -8,14 +8,22 @@ import { SuiteLocal } from './SuiteLocal';
 
 describe('MSBuild Generator', () => {
   const local = new SuiteLocal();
-  const fs = local.fs;
 
   after(local.after.bind(local));
 
-  it('Generates locations in order', async () => {
+  it('Generates roots with a trailing slash', () => {
```

Comment:
```suggestion
  it('Generates roots without a trailing slash', () => {
```
Or am I misreading the test?